pkg/forth/forth.go

```diff
@@ -183,13 +184,18 @@ func (f *stack) Empty() bool {
 // otherwise.
 func errRecover(errp *error) {
 	e := recover()
+	Debug("pkg/forth:e is %v:%T", e, e)
 	if e != nil {
-		if _, ok := e.(runtime.Error); ok {
-			Debug("errRecover panics with a runtime error")
+		switch err := e.(type) {
+		case runtime.Error:
+			Debug("pkg/forth:errRecover panics with a runtime error")
 			panic(e)
+		case error:
+			*errp = fmt.Errorf("%w", err)
+		default:
+			*errp = fmt.Errorf("pkg/forth:%v", err)
```

Comment:
and %v -> %w here

Reply:
can't do that here, it's not an error, it's any